aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md

```diff
@@ -753,44 +753,44 @@ You can now add JavaScript files to a central list that will be implicitly inclu
 
 The IntelliSense list for CSS now filters based on the CSS properties and values supported by the selected schema.
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image23.png)
+![Screenshot that shows the IntelliSense list for C S S now filters based on the C S S properties and values supported by the selected schema.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image23.png)
 
 IntelliSense also supports title case searches:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image24.png)
+![Screenshot shows that IntelliSense also supports title case searches.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image24.png)
 
 <a id="_Toc318097417"></a>
 #### Hierarchical indentation
 
 The CSS editor uses indentation to display hierarchical rules, which gives you an overview of how the cascading rules are logically organized. In the following example, the #list a selector is a cascading child of list and is therefore indented.
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image25.png)
+![Screenshot that shows an example of the #list selector as a cascading child of list and is therefore indented.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image25.png)
 
 The following example shows more complex inheritance:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image26.png)
+![Screenshot that shows an example with more complex inheritance.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image26.png)
 
 The indentation of a rule is determined by its parent rules. Hierarchical indentation is enabled by default, but you can disable it the Options dialog box (Tools, Options from the menu bar):
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image27.png)
+![Screenshot that shows the indentation of a rule, which is determined by its parent rules. Hierarchical indentation is enabled by default, but you can disable it the Options dialog box (Tools, Options from the menu bar).](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image27.png)
```

Comment:
is there a preposition missing in "but you can disable it the Options dialog box"? I suppose it was copy pasted from the content in which case that should be ok.